crates/libs/windows/src/lib.rs

```diff
@@ -13,11 +13,9 @@ extern crate self as windows;
 pub mod core {
     pub use windows_core::*;
 
-    #[doc(hidden)]
     #[cfg(feature = "implement")]
     pub use windows_implement::implement;
 
-    #[doc(hidden)]
     #[cfg(feature = "implement")]
     pub use windows_interface::interface;
 }
```

Comment:
Now that is cheeky :grin: and explains why I was trying to find an `implement` reexport in the `windows-core` crate.

Not to be a nuisance, but I wouldn't have done it this way in favour of not tricking people that `implement`/`interface` exist in the `windows-core` crate, whenever they might want to migrate from this `windows` crate. Instead, how about:

```rust
pub use windows_core as core;
pub use windows_implement::implement;
pub use windows_implement::interface;
```

?

Reply:
These were originally in the `windows::core` module so this was just done for compatibility.

Reply:
Are there plans to break that backwards-compat at some point?